var/spack/repos/builtin/packages/py-ray/package.py

```diff
@@ -42,3 +43,15 @@ class PyRay(PythonPackage):
     depends_on('py-redis@3.3.2:3.4', type=('build', 'run'))
     depends_on('py-opencensus', type=('build', 'run'))
     depends_on('py-prometheus-client@0.7.1:', type=('build', 'run'))
+    depends_on('py-setproctitle', type=('build', 'run'))
```

Comment:
Can you add a comment saying that the `build` function in `setup.py` will automatically download and install these dependencies if they aren't found?

Reply:
I added a couple of comment block that describe the two kind of issues encountered, hopefully they are not misleading.